web-gateway/app/controllers/ItemController.java

```diff
@@ -30,19 +32,25 @@
     private final ItemService itemService;
     private final BiddingService bidService;
 
+    private Configuration config;
```

Comment:
It isn't necessary to store the configuration in an instance variable, because it's only used inside the constructor.